horovod/spark/lightning/datamodule.py

```diff
@@ -49,7 +50,10 @@ def __init__(self, train_dir: str, val_dir: str, num_train_epochs: int=1, has_va
     def setup(self, stage=None):
         # Assign train/val datasets for use in dataloaders
         if stage == 'fit' or stage is None:
-            transform_spec = TransformSpec(self.transformation) if self.transformation else None
+            if self.transformation is None and self.transformation_edit_fields is None:
+                transform_spec = None
+            else:
+                transform_spec = TransformSpec(func=self.transformation, edit_fields=self.transformation_edit_fields)
```

Comment:
Can you add a comment of linking `TransformSepc` constructor usage from Petastorm?
https://github.com/uber/petastorm/blob/3f248003221a648261a36189c95c8705f6ef34ad/petastorm/transform.py#L27

Reply:
Sure. I'll add param for removed_fields and selected_fields then :)

Reply:
Since only one of removed_fields and selected_fields can be specified, I'll go with removed_fields as if you don't return a certain field, it echoes errors, but keeping extra fields should be fine.